components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -1,6 +1,6 @@
 <chef-page
   subheading="Resources that match the type and all conditions are included in the project."
-  [attr.confirm-btn-text]=" saving ? 'Saving Rule...' : 'Save Rule' "
+  [attr.confirm-btn-text]="(authorizedChecker.isAuthorized$ | async) ? saving ? 'Saving Rule...' : 'Save Rule' : ''"
```

Comment:
Because `<chef-page>` is a stencil component, rather than increase the surface area of the interface (passing parameters to use an actuall `<app-authorized>` within `<chef-page>`), just taking advantage of an existing feature--set the button label to empty if not authorized.